packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -655,6 +655,19 @@ class OverflowBox extends OneChildRenderObjectWidget {
       ..maxHeight = maxHeight
       ..alignment = alignment;
   }
+
+  @override
```

Comment:
My understanding is that we can use `@override` when https://github.com/flutter/flutter/issues/1220 is possible.